client/src/components/home-layout/style.scss

```diff
@@ -0,0 +1,5 @@
+
+.channel-detail {
```

Comment:
@rollaball , correct me if I'm wrong, wouldn't it be better not to rely on `scss` at all? Might be better if we use this within the components itself.

Reply:
Updated